lib/themes-source/base.less

```diff
@@ -30,7 +30,7 @@
  * The picker input element.
  */
 .picker__input {
-    cursor: default;
```

Comment:
@bl4ckdu5t @ivandoric that's not the expected behavior: http://www.wufoo.com/html5/types/4-date.html

Date inputs are not text entry and so the caret pointer makes no contextual sense. The current way is the way browsers handle native date inputs.